modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/streaming/BytesStreamingExtensionTestCase.java

```diff
@@ -203,7 +203,7 @@ private ParameterModel getStreamingStrategyParameterModel(Supplier<Parameterized
 
   private ConfigurationModel getConfigurationModel() {
     return getExtensionModel("Marvel")
-        .map(extension -> extension.getConfigurationModel("dr-strange").get())
+        .map(extension -> extension.getConfigurationModel("dr-strange-config").get())
```

Comment:
doesn't this mean that you should change the marvel extension test json as well?